developer/web-api/new-tracker.md

```diff
@@ -1312,7 +1328,7 @@ You can use a range of operators for the filtering:
 
 The `JSON` response can look like the following.
 
-Please note that field filtering (`fields=...`) support is planned but not yet implemented.
+Since version 2.37 responses can be filtered on desired fields, see [Request parameter to filter responses](#Request-parameter-to-filter-responses)
```

Comment:
I don't think we should be mentioning specific versions, since docs are already versioned

Reply:
fair enough